src/admin/menu/menu.const.ts

```diff
@@ -0,0 +1,24 @@
+import { OptionsType } from 'react-select';
+
+import { IconName } from '@viaa/avo2-components';
+
+import { ROUTE_PARTS } from '../../shared/constants';
+import { ReactSelectOption } from '../../shared/types';
+
+export const MENU_PATH = {
```

Comment:
is MENU_PATHS niet logsicher dan MENU_PATH
anders verwacht ik eerder een string ipv een dictionary

Reply:
Zou de inner properties ook veranderen naar ROOT, DETAIL, CREATE, EDIT. Anders heb je herhaling van MENU in `MENU_PATHS.MENU_DETAIL`.

Reply:
@bertyhell ik zie die PATH objecten een beetje als string enums en ik volg daar gwn hoe typescript die declareert in hun docs, maar ik zou ze kunnen aanpassen
@benjaminnaesen ik doe dat zo omdat ik in de root van admin al die paths spread in `ADMIN_PATH` om te vermijden dat je veel imports gaat doen in pagina's waar je deze nodig hebt

Reply:
ok, laat maar als PATH dan